src/pages/waiting-room/waiting-room.ts

```diff
@@ -35,12 +36,18 @@ export class WaitingRoomPage extends BasePageComponent {
     public navParams: NavParams,
     public platform: Platform,
     public authenticationProvider: AuthenticationProvider,
+    private deviceProvider: DeviceProvider,
   ) {
     super(platform, navCtrl, authenticationProvider);
   }
 
   ionViewDidEnter(): void {
     this.store$.dispatch(new waitingRoomActions.WaitingRoomViewDidEnter());
+    this.deviceProvider.enableSingleAppMode();
```

Comment:
Could you write some tests so we can made sure these are called. 

Example Tests can be found in test-report.spec.ts

You will need to update the DeviceProvider Mock so it has the correct functions & uses jasmine mocks, see authentication.mock.ts for examples of those